pyneuroml/plot/PlotSpikes.py

```diff
@@ -106,18 +102,7 @@ def process_args():
     return parser.parse_args()
 
 
-def main(args=None):
-    if args is None:
-        args = process_args()
-    run(a=args)
-
-
-def read_sonata_spikes_hdf5_file(file_name: str):
-    """Read a sonata HDF5 file with spike data
```

Comment:
Please keep the docstrings. Every function should have one. This is required for the API docs:

https://pyneuroml.readthedocs.io/en/development/